Dockerfile.cpu

```diff
@@ -44,7 +44,7 @@ COPY scripts/ scripts/
 RUN find . | grep -E "(__pycache__|\.pyc|\.pyo$)" | xargs rm -rf
 
 RUN /bin/bash -c 'source activate allennlp && INSTALL_TEST_REQUIREMENTS="true" ./scripts/install_requirements.sh'
-RUN /bin/bash -c 'source activate allennlp && pip install --no-cache-dir -q http://download.pytorch.org/whl/cu75/torch-0.1.12.post2-cp35-cp35m-linux_x86_64.whl'
+RUN /bin/bash -c 'source activate allennlp && pip install --no-cache-dir -q http://download.pytorch.org/whl/cu80/torch-0.2.0.post1-cp35-cp35m-manylinux1_x86_64.whl'
```

Comment:
the gpu one uses the cuda base image, the cpu one uses a plain ubuntu base image. (previously they were pointing at different pip wheels, as per the pytorch documentation, but that might not be true in pytorch 2.0)